Source/qol.cpp

```diff
@@ -12,10 +12,25 @@
 namespace devilution {
 namespace {
 
+namespace xpbar {

```

Comment:
I think it'd be cleaner to move the xpbar stuff to its own file. This one is growing unwieldy.

Reply:
Perhaps `Source/qol/xpbar.cpp` /cc @AJenbo

Reply:
I see no reason for the "qol" in the name. It is already under a toggle, the file itself doesn't need the extra info.

Reply:
All for splitting the qol stuff up, but to be honest this would inflate the PR even further. So I'd suggest doing this in a separate commit.

Reply:
> Perhaps `Source/qol/xpbar.cpp` /cc @AJenbo

I was just thinking of that earlier today.

Reply:
> All for splitting the qol stuff up, but to be honest this would inflate the PR even further. So I'd suggest doing this in a separate commit.

I think that is exactly what is being suggestd.